crates/pg_lsp_new/src/main.rs

```diff
@@ -0,0 +1,36 @@
+use std::{fs::File, path::PathBuf, str::FromStr};
```

Comment:
what benefit does this have over the pg_cli?

Reply:
I wasn't able to start the extension in vscode with `pg_cli lsp-proxy`, it throws an ENOENT error...

But I'm with you, we shouldn't have a separate main here. I only wanted to use this for debugging, then remove it and fix the extension so it uses the cli in a separate pr 👍

Reply:
ahh saw this comment too late. is this now fixed?